File: APIProject/APIProject/Models/Booking.cs
using System.ComponentModel.DataAnnotations;

namespace APIProject.Models
{
    public class Booking
    {
        public string Id { get; set; } = Guid.NewGuid().ToString();
        [Required]
        public string UserId { get; set; }
        [Required]
        public string BusId { get; set; }
        [Required]
        public string FromLocationId { get; set; }
        [Required]
        public string ToLocationId { get; set; }
        [Required]
        public string[] SeatNos { get; set; }
        public int TotalPrice { get; set; }
    }
}

File: RedBusApp/src/app/models/class/Booking.ts
export class Booking{
    id: string;
    userId: string;
    busId: string;
    fromLocationId: string;
    toLocationId: string;
    seatNos: string[];
    totalPrice: number;
    constructor(){
        this.id = "";
        this.userId = "";
        this.busId = "";
        this.fromLocationId = "";
        this.toLocationId = "";
        this.seatNos = [];
        this.totalPrice = 0;
    }
}